python/FindPythonAnaconda.cmake

```diff
@@ -61,15 +61,16 @@ if(NOT CMAKE_FIND_ANACONDA_PYTHON_INCLUDED)
     set( _py_version_patch ${CMAKE_MATCH_3} )
     set( ANACONDA_PYTHON_VERSION ${_py_version_major}.${_py_version_minor} )
 
-    
-    if( ${_py_version_major} MATCHES 2 )
+
+    if( (${_py_version_major} MATCHES 2) OR
+        (${_py_version_major} MATCHES 3 AND ${_py_version_minor} MATCHES 8) )
```

Comment:
Is this a change moving forward? If so should this be major >= 3 and minor >= 8?

Reply:
I'm not sure I can look into this

Reply:
This is dependent upon how the Python distribution was compiled. For 3.8, Anaconda compiled it without pymalloc, hence no m, but we don't know how they will compile 3.9 so we'll have to deal with that when 3.9 becomes the standard for Anaconda.

Reply:
Will this error in a not silent way if _py_ext is incorrect?

Reply:
no, it will fail to link while building